### PR TITLE
Odin simulation workflow loader and hardcoded choppers.

### PR DESCRIPTION
**Pull request by YooSunYoung**
Related to #20, #26  and Related to [one of milestone for p6](https://github.com/scipp/essimaging/milestone/3)

The notebook is for P6 milestone report so it is expected to be refactored and integrated into a module as a reusable workflow.

#56 should be merged first.

### CODE REVIEW

File: docs/user-guide/odin_simulation.ipynb [hunk 1]
```diff
@@ -0,0 +1,1049 @@
...
+    ") -> sc.Variable:\n",
+    "    z = sc.zeros_like(x) + z_pos\n",
+    "    var = (\n",
+    "        sc.concat([x, y, z], 'event')\n",
+    "        .fold('event', dims=['pos', 'event'], shape=[3, len(x)])\n",
+    "        .transpose(dims=['event', 'pos'])\n",
+    "        .values\n",
+    "    )\n",
+    "    return sc.vectors(dims=['event'], values=var, unit='m')\n"
+   ]
```
Comment (nvaytet) [inline docs/user-guide/odin_simulation.ipynb:262]: I feel that most of the code at the start of this notebook should be part of a submodule in the package, and not in the notebook?

File: docs/user-guide/odin_simulation.ipynb [hunk 2]
```diff
@@ -0,0 +1,1049 @@
...
+    "    )\n",
+    "    y_range = sc.array(\n",
+    "        dims=['y'],\n",
+    "        values=[artist.xy[0], artist.xy[0] + artist.width],\n",
+    "        unit=figure.canvas.units['y'],\n",
+    "    )\n",
+    "    return RectInfo(x_range.min(), x_range.max(), y_range.min(), y_range.max())\n",
+    "\n",
+    "\n",
+    "class MergedRectanglesTool(ToggleTool):\n",
```
Comment (nvaytet) [inline docs/user-guide/odin_simulation.ipynb:481]: I also think this code should go in a submodule, so we can just simply import the tool in the notebook?

File: docs/user-guide/odin_simulation.ipynb (hunk 1 above)
Comment (YooSunYoung) [reply docs/user-guide/odin_simulation.ipynb:262]: Yeah, as Søren wanted everything before December,
I thought we don't have time to make them as a module, write test and documentation.

Also, McStas format is not stable at the moment so I didn't want to start working on a mcstas IO module...

File: docs/user-guide/odin_simulation.ipynb (hunk 2 above)
Comment (YooSunYoung) [reply docs/user-guide/odin_simulation.ipynb:481]: Yeah. Then I'll do that in plopp maybe...?

File: docs/user-guide/odin_simulation.ipynb (hunk 1 above)
Comment (nvaytet) [reply docs/user-guide/odin_simulation.ipynb:262]: ok

**Review by nvaytet — APPROVED** [review]

**Pull request merged**